godot-ffi/src/interface_init.rs

```diff
@@ -112,9 +112,19 @@ pub fn ensure_static_runtime_compatibility(
     }
 }
 
+#[cfg(before_api = "4.5")]
+type GDExtensionInterfaceGetGodotVersion = sys::GDExtensionInterfaceGetGodotVersion;
+#[cfg(since_api = "4.5")]
+type GDExtensionInterfaceGetGodotVersion = sys::GDExtensionInterfaceGetGodotVersion2;
```

Comment:
Please don't prefix anything we define with `GDExtension`. 

If we do this in other places, we should fix it.